src/builders/utils/uploader.ts

```diff
@@ -29,3 +39,62 @@ export async function uploadBuildToS3(ctx: IUploadCtx) {
     return fileLocation;
   }
 }
+
+export async function uploadBuildToTestFlight(ctx: IContext, options: IJobOptions, bundleIdentifier?: string) {
+  const fastlaneEnvVars = {
+    FASTLANE_USER: options.appleId,
+    FASTLANE_SKIP_UPDATE_CHECK: 1,
+    FASTLANE_DISABLE_COLORS: 1,
+    CI: 1,
+    LC_ALL: 'en_US.UTF-8',
+    FASTLANE_PASSWORD: options.applePassword,
+    FASTLANE_APPLE_APPLICATION_SPECIFIC_PASSWORD: options.appSpecificPassword,
+  };
+
+  const produceArgs = [
+    'produce',
+    '--username',
+    options.appleId,
+    '--app_identifier', // bundle id
+    bundleIdentifier,
+    '--app_name',
+    options.appName,
+    '--app_version',
+    '1.0',
```

Comment:
Why `1.0`? Is this upload feature meant to only upload one version of an application to Testflight? If no then what's the way of making sure a user builds and uploads the app only once?